src/context/mod.rs

```diff
@@ -313,6 +316,11 @@ pub trait Context<'a>: ContextInternal<'a> {
         JsBuffer::new(self, size)
     }
 
+    #[cfg(feature = "napi-5")]
+    fn date(&mut self, value: impl Into<f64> + PartialOrd) -> Result<Handle<'a, JsDate>, DateError> {
```

Comment:
Why does this need to be `PartialOrd`? Shouldn't we convert to `f64` before performing a comparison?